nvflare/job_config/base_app_config.py

```diff
@@ -62,6 +63,12 @@ def add_ext_script(self, ext_script: str):
 
         self.ext_scripts.append(ext_script)
 
+    def add_ext_dir(self, ext_dir: str):
```

Comment:
Please add docstring explaining what this method does and how it's supposed to be used.

Reply:
This is the internal implementation place. The docstring is added to the add_external_dir() infed_job.py